delta_Go/check_submission.py

```diff
@@ -0,0 +1,36 @@
+import time
+from pathlib import Path
+from typing import Callable
+
+import delta_utils.check_submission as checker
+import numpy as np
+from torch import nn
```

Comment:
Interestingly, my pre-commit wants to move this to next to the `game_mechanics` install...